File: SeatMe-Frontend/src/app/model/Restaurant.ts
export class Restaurant {
  id: number;
  name: string;
  address: string;
  contactNumber: string;
  managerEmail: string;
  managerPassword: string;
  imageUrl:string

  constructor(
    id: number,
    name: string,
    address: string,
    contactNumber: string,
    managerEmail: string,
    managerPassword: string,
    imageUrl:string
  ) {
    this.id = id;
    this.name = name;
    this.address = address;
    this.contactNumber = contactNumber;
    this.managerEmail = managerEmail;
    this.managerPassword = managerPassword;
    this.imageUrl=imageUrl;
  }
}

File: SeatMe-Backend/src/main/java/edu/icet/ecom/dto/Restaurant.java
package edu.icet.ecom.dto;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Restaurant {
    private Long id;
    private String name;
    private String address;
    private String contactNumber;

    private String managerEmail;
    private String managerPassword;
    private String imageUrl;
}
